antenna/throttler.py

```diff
@@ -113,11 +113,11 @@ def throttle(self, raw_crash):
                 if rule.percentage == 100 or (random.random() * 100.0) <= rule.percentage:  # nosec
                     response = ACCEPT
                 else:
-                    response = DEFER
+                    response = REJECT
```

Comment:
This is the big change. Instead of "ACCEPT if we roll a good number and DEFER if we don't", we now "ACCEPT if we roll a good number and REJECT if we don't".

Reply:
This would be better codified as part of the rule structure. I'll look at doing that now.